common/transport/http/src/rest_api_client.ts

```diff
@@ -22,6 +22,10 @@ export interface HttpTransportError extends Error {
   responseBody?: any;
 }
 
+export interface HttpOperation {
```

Comment:
that feels like a really incomplete interface for an HttpOperation or a really generic name for something that is just aborting a request.